src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2257,4 +2257,17 @@ limitations under the License.
             </include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>Cooktop</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Cooktop</typeName>
+        <class>Simple</class>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0078</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="On/Off" client="false" server="true" clientLocked="true" serverLocked="true">
```

Comment:
The other device types also seem to include the descriptor cluster. Which seems fairly repetitive to me, tbh, but it seems like it might be needed since it's all over this file.

Reply:
Ok sounds good. Will put that in there